docs/painless/painless-contexts/painless-field-context.asciidoc

```diff
@@ -26,3 +26,64 @@ a customized value for each document in the results of a query.
 *API*
 
 The standard <<painless-api-reference, Painless API>> is available.
+
+
+*Example*
+
+To run this example, first follow the steps in
+<<painless-context-examples, context examples>>.
+
+We will use two scripts to create two additional fields for each search hit.
+These script fields will output computed custom information.
+
+The first script calculates the day of the week for the field `datetime`:
+
```

Comment:
The callout below is fairly redundant. I'd remove the callout and edit the intro to read:

The first script gets the doc value for the `datetime` field and calls the `getDayOfWeek` function to determine the corresponding day of the week.